packages/flame/lib/src/effects/move_along_path_effect.dart

```diff
@@ -85,7 +85,8 @@ class MoveAlongPathEffect extends MoveEffect {
         'An `oriented` MoveAlongPathEffect cannot be applied to a target that '
         'does not support rotation',
       );
-      (target as AngleProvider).angle = _lastAngle = -start.angle;
+      _lastAngle = -start.angle;
+      (target as AngleProvider).angle += -start.angle;
```

Comment:
I don't think this is correct, since `followAngle` is true, it should be following the angle of the effect, and not be affected by what its current angle is?

Reply:
Well the problem is with the preexisting angle of the component, in the current release this is always initialized to the tangent. It should keep consider the angle of the component itself. Please have a look to the demo in the issue.

Reply:
Then you should be using `nativeAngle` to set that, not `angle`.

Reply:
Sounds good, we can close this as intended behaviour